mobile/src/full/java/org/openhab/habdroid/core/CloudMessagingHelper.kt

```diff
@@ -70,4 +72,15 @@ object CloudMessagingHelper {
         }
         else -> context.getString(R.string.info_openhab_gcm_connected)
     }
+
+    // Used to clear all notifications from statusbar
+    fun clearAllNotifications(context: Context) {
+        // to clear the notifications on current device
+        val notificationManager: NotificationManager =
+            context.getSystemService(NOTIFICATION_SERVICE) as NotificationManager
+        notificationManager.cancelAll()
```

Comment:
As mentioned before: This is wrong. We have non-FCM notifications as well, and we must not clear those.

Reply:
So should I check whether it's a FCM notification and then clear?

Reply:
I came to we can use `n.groupKey?.endsWith("gcm")` to identify fcm notifications after API 23. How can I do the same for prior APIs ?

Reply:
AFAICT there is no way to implement this on API < 23.